### ISSUE TITLE
Error/Bug with the slappers commands

### ISSUE THREAD

**Issue by EduGH7** [jojoe77777/Slapper#201]
Sometimes when I start the server like always and I hit the slapper the commands work good, but if I close the server and start it again, or I restart the server the slapper commands don't work but I check the commands list of the slapper and there are like always

**Comment by benda95280** [comment]
Same here, with the build 67.
It seems it occure when the server crash or bad closed.

**Comment by JuniorYT** [comment]
Please go into your plugins and update it! Simple fix it needs to be updated they updated this morning!

**Comment by EduGH7** [comment]
With the 1.5.0 it's fixed?

**Comment by benda95280** [comment]
Just try to update it, seems it need new pocketmine too.
After update of pocketmine, no more slapper in the world.
I've reverted back to my previous version.

**Comment by EduGH7** [comment]
Yep, the slappers dissapear, but with the new pocketmine I have problems with my economyshop plugin and apparently there is no fix yet .-.

**Comment by Gianluxx** [comment]
The slappers disappeared because you started the server without the slapper plugin. Spawn them again now

**Comment by EduGH7** [comment]
I update and now I have slapper 1.5.0 and I got this same problem

**Comment by benda95280** [comment]
It might be due to other plugin ? If i reload the server, it works.
I've read somewhere some change have been done to have plugin loaded in random Order. Could it be that why somestimes it works ?

**Comment by EduGH7** [comment]
I tried to don't close the server or restart it externally, I only use the command /reload and apparently it works

**Comment by AidanSpeakss** [comment]
Did anyone find a fix for this?

**Comment by benda95280** [comment]
Nope, i have not found time to do more test. People having the problem could post their plugin list ?

**Comment by SleepSpace9** [comment]
It's working for me since I've added `load: STARTUP` into plugin.yml of the Slapper plugin, because it always didn't work if Slapper was loaded too late. But I guess it depends on everyones plugins.

**Comment by Johnsonk79** [comment]
I figured out why, Its because your world is not properly loaded, most likely because its a void generated world, to fix, you will have to download nbt explorer and change the generator of the world (In level.dat of the world file) you have problem with from void to normal

**Comment by benda95280** [comment]
The answer from SleepSpace9 seems to be the right one.

**Comment by Johnsonk79** [comment]
Maybe his will work for some people, but mine worked for me and my friends, who knows
